### PR TITLE
README: fix build instructions

### PR DESCRIPTION
**Pull request by starius**
The list of ghcup dependencies was not complete. I copied the full list from https://www.haskell.org/ghcup/install/#version-12

Qt installation command didn't work. I fixed it.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,10 +29,12 @@ ghcup set cabal 3.10.3.0
 #### For QT:
 
 ```bash
-sudo apt-get install qtdeclarative5-dev && \
```
Comment (prolic) [inline README.md:32]: good catch for removing the "&&"

**Comment by prolic** [comment]
I maybe should have added those are build instructions for Ubuntu / Linux Mint. I can't (and don't want to) maintain exact instructions for each and every linux distro out there.
But I can build a docker image that let's you build this app on any linux machine, would that help?

If not, I'd suggest splitting this into:

Ubuntu Setup / Debian Setup / (maybe Docker Setup)

I hope this makes sense.

**Comment by starius** [comment]
I would very appreciate if Dockerfile is provided!

**Comment by prolic** [comment]
done via 297a9f6983b5179632566623dbd8574a683bf385

**Pull request closed without merging**